modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -108,6 +108,7 @@
                 },
                 "claimant": {
                   "type": "object",
+                  "description": "If the claimant is a non-Veteran, claimant information is required. Without this information, we will be unable to identify the claimant. If you're a healthcare provider or an employee of a healthcare provider requesting review of a VA payment decision, you must identify the healthcare provider as the claimant and complete all relevant information in the claimant object.",
```

Comment:
I think we'd be safe to add this description to all our schemas that include claimant now, so that we don't forget to later.